### ISSUE TITLE
extension icon disappear in wsl2

### ISSUE THREAD

**Issue by Direct-A** [microsoft/vscode-remote-release#3084]
- VSCode Version: 1.45.1
- OS Version: Windows_NT x64 10.0.19041.264
			  wsl2 ubuntu 18.04.4
			  kernal 4.19.84-microsoft-standard

Steps to Reproduce:

1. When connect to wsl, and all extention loaded.

in stable
![stable](https://user-images.githubusercontent.com/46991254/83326868-e4cc7a00-a2a9-11ea-810b-ff82a0898f50.png)

in insiders it happends too
![insider](https://user-images.githubusercontent.com/46991254/83326916-4a206b00-a2aa-11ea-8d41-ea2281339203.png)

Does this issue occur when all extensions are disabled?: 

  No, there is no extensions icons.

**Comment by sandy081** [comment]
It seems all custom view container icons are not visible which are svgs.

Can you please do following: 
- open Dev tools using command `Developer > Toggle Developer Tools`
- in the `Elements` tab use the inspect element action 
- Click on the location where icon is missing
- Share the screenshot of elements and style as shown below

<img width="771" alt="image" src="https://user-images.githubusercontent.com/10746682/83533249-9b6d6c00-a4ef-11ea-9fea-63d76ff5a21c.png">

**Comment by Direct-A** [comment]
## in stable
### left
![stable-1](https://user-images.githubusercontent.com/46991254/83548701-c4fdb600-a536-11ea-9f42-31c92fd2b78c.png)
![stable-2](https://user-images.githubusercontent.com/46991254/83548713-cb8c2d80-a536-11ea-806e-5920148c2e89.png)
![stable-3](https://user-images.githubusercontent.com/46991254/83548716-ccbd5a80-a536-11ea-9968-721d0cb689eb.png)

### top
![stable-a](https://user-images.githubusercontent.com/46991254/83548874-0db56f00-a537-11ea-8b9e-a238607c7994.png)
![stable-b](https://user-images.githubusercontent.com/46991254/83548894-1148f600-a537-11ea-8985-69bfa5063ece.png)
![stable-c](https://user-images.githubusercontent.com/46991254/83548896-1312b980-a537-11ea-85d0-77b64e0c4118.png)

---
## in insiders
### left
![insiders-1](https://user-images.githubusercontent.com/46991254/83550156-f8d9db00-a538-11ea-8cf3-122391d38983.png)
![insiders-2](https://user-images.githubusercontent.com/46991254/83550163-fc6d6200-a538-11ea-9596-8b84dbcd108c.png)
![insiders-3](https://user-images.githubusercontent.com/46991254/83550171-01caac80-a539-11ea-8f02-9af908b99834.png)

### top
![insiders-a](https://user-images.githubusercontent.com/46991254/83550210-1149f580-a539-11ea-85b4-b2af345d3773.png)
![insiders-b](https://user-images.githubusercontent.com/46991254/83550220-15761300-a539-11ea-9261-e215fc4b1f44.png)
![insiders-c](https://user-images.githubusercontent.com/46991254/83550224-17d86d00-a539-11ea-9cec-35abab296443.png)

**Comment by sandy081** [comment]
It seems not just activity bar icons are missing, other icons from remote extensions are missing.

Can you please check if the workaround mentioned here works for you - https://github.com/microsoft/vscode/issues/92323#issuecomment-611045838

**Comment by Direct-A** [comment]
It doesn't work, I have ckecked `HKEY_CLASSES_ROOT\.svg`, there is no items of ConceptDraw. And I haven't install this software.
I have tried to delete some of them, but it doesn't work.

this is what in the `HKEY_CLASSES_ROOT\.svg`
![Snipaste_2020-06-04_10-25-07](https://user-images.githubusercontent.com/46991254/83707917-bbb33d00-a64d-11ea-8e52-b299b27ed7f4.png)

and I tried to open the network tab, this is what I got
![Snipaste_2020-06-04_11-52-34](https://user-images.githubusercontent.com/46991254/83713296-44d07100-a65a-11ea-8fe1-e52162d9dff4.png)
